boards/riscv/hpm6750evkmini/hpm6750evkmini.dts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 hpmicro
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <hpmicro/hpm6750.dtsi>
+#include "hpm6750evkmini-pinctrl.dtsi"
+
+/ {
+	model = "HPMicro HPM6750EVKMINI";
+	compatible = "HPMicro,hpm6750evkmini";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &sram;
+		zephyr,itcm = &ilm;
+		zephyr,dtcm = &dlm;
+	};
+
+	core0_lm_slv: memory@1000000 {
+		device_type = "memory";
+		reg = <0x1000000 DT_SIZE_K(512)>;
+	};
+
+	core1_lm_slv: memory@1180000 {
+		device_type = "memory";
+		reg = <0x1180000 DT_SIZE_K(256)>;
+	};
+
+	noncache: memory@11c0000 {
+		device_type = "memory";
+		reg = <0x11c0000 DT_SIZE_K(256)>;
+	};
+
+	dram: memory@40000000 {
+		device_type = "memory";
+		reg = <0x40000000 DT_SIZE_M(16)>;
+	};
```

Comment:
does this really belong to board level DTS?

Reply:
All changes required except dram